awscli/autoprompt/widgets.py

```diff
@@ -48,11 +50,32 @@ def apply_transformation(self, text_input):
         return Transformation(fragments)
 
 
+class TitleLine:
+
+    def __init__(self, title):
+        fill = partial(Window, style='class:frame.border')
+        self.container =  VSplit([
+            fill(char=Border.HORIZONTAL),
+            fill(width=1, height=1, char='|'),
+            # Notice: we use `Template` here, because `self.title` can be an
+            # `HTML` object for instance.
+            Label(lambda: Template(' {} ').format(title),
+                  style='class:frame.label',
+                  dont_extend_width=True),
```

Comment:
Do we actually care about it being an `HTML` object? Could we just specify `title` here instead of the lambda?